pkg/apps/application.css

```diff
@@ -137,3 +137,8 @@
 .right-menu * {
     display: inline-block;
 }
+
+/* Give it the same padding as button have */
```

Comment:
Hmm PF4 buttons do not have margin - they do have padding though.

padding: var(--pf-c-button--PaddingTop) var(--pf-c-button--PaddingRight) var(--pf-c-button--PaddingBottom) var(--pf-c-button--PaddingLeft);

Since you achieve the visual fix with this as well I am ok to keep this one as well of course.